chart/mixnode/templates/_helpers.tpl

```diff
@@ -0,0 +1,63 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
removed fullnameOverride, others are being used.

Reply:
I mean in general, for instance, if we are not using `tolerations`, `affinity` or `nodeSelectors` better not define the values and have a more clean statefulset, wdyt?

Reply:
removed tolerations n affinity, we are using node selectors.

Reply:
i think it would be better to not use node selectors as discussed below, we can't be sure that a single node (always the same) is always tagged with the nym tag. Goes against cloud native principles, the nodes are catle (bigger catle than containers but castle after all) and their IPs can't be taken for granted. Instead we should rely on the IP of the load balancer associated with our service.